hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainerData.java

```diff
@@ -384,6 +413,10 @@ public String getDeletingBlockKeyPrefix() {
     return formatKey(DELETING_KEY_PREFIX);
   }
 
+  public String getFinalizeBlockKey() {
+    return formatKey("");
```

Comment:
missing a key here?

Reply:
I wanted to keep only containerId as key, [formatKey](https://github.com/apache/ozone/blob/cd0c55e4ffa879b3e7cffe182237f0e1c824a7c0/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainerData.java#L429) returns key as containerId in v3, so that would work.
But because of this [comment](https://github.com/apache/ozone/pull/5783#discussion_r1429729146), changed now key format to use containerId|localID